bots/tests-scan

```diff
@@ -458,10 +360,10 @@ def cockpit_tasks(api, update, branch_contexts, repo, pull_data, pull_number, sh
             return False
 
         if trigger_externals():
-            for proj_repo in REPO_BRANCH_CONTEXT:
+            for proj_repo in testmap.projects():
                 if proj_repo == "cockpit-project/cockpit":
                     continue
```

Comment:
Looking forward to the day when this can go away, and cockpit is "just one project" :-)

Reply:
:)